qiskit/circuit/library/standard_gates/equivalence_library.py

```diff
@@ -616,6 +616,29 @@
     def_ecr.append(inst, qargs, cargs)
 _sel.add_equivalence(ECRGate(), def_ecr)
 
+# ECRGate decomposed to Clifford gates (up to a global phase)
+#
+#                  global phase: 7π/4
+#      ┌──────┐         ┌───┐                 ┌───┐
+# q_0: ┤0     ├    q_0: ┤ S ├──────────────■──┤ X ├
+#      │  Ecr │  ≡      ├───┤┌─────┐┌───┐┌─┴─┐├───┤
+# q_1: ┤1     ├    q_1: ┤ H ├┤ Sdg ├┤ H ├┤ X ├┤ X ├
+#      └──────┘         └───┘└─────┘└───┘└───┘└───┘
+
+q = QuantumRegister(2, "q")
+def_ecr_cliff = QuantumCircuit(q, global_phase=-pi / 4)
+for inst, qargs, cargs in [
+    (SGate(), [q[0]], []),
+    (HGate(), [q[1]], []),
+    (SdgGate(), [q[1]], []),
+    (HGate(), [q[1]], []),
+    (CXGate(), [q[0], q[1]], []),
+    (XGate(), [q[0]], []),
+    (XGate(), [q[1]], []),
+]:
+    def_ecr_cliff.append(inst, qargs, cargs)
+_sel.add_equivalence(ECRGate(), def_ecr_cliff)
```

Comment:
You used a slightly more efficient decomposition (at least in terms of 1q gate count) in the `Clifford` class itself.  Perhaps using the `SXdg` form here is slightly neater; the basis translator will be able to convert `sxdg` to `h . sdg . h` as appropriate if needed, and your changes to the Clifford class mean that that form would already be all-Clifford.